include/RED4ext/Scripting/Natives/AnimKeyFrames.hpp

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <cstdint>
+
+namespace RED4ext
+{
+namespace anim
+{
+enum KeyFrameComponent
+{
+    Translation = 0,
+    Rotation,
+    Scale,
+};
+
+struct KeyFrameHeader
+{
+    uint16_t timeNormalized;     // 00
+    uint16_t boneIndex : 13;     // 02
+    uint16_t componentType : 2;  // 0E
+    uint16_t rotationWSign : 1;  // 0F
+};
+RED4EXT_ASSERT_SIZE(KeyFrameHeader, 0x4);
+
+struct KeyFrameCompressed : KeyFrameHeader
+{
+    uint16_t x; // 00
+    uint16_t y; // 02
+    uint16_t z; // 04
+};
+RED4EXT_ASSERT_SIZE(KeyFrameCompressed, 0xA); // total struct size is header size + struct size
```

Comment:
```suggestion
RED4EXT_ASSERT_SIZE(KeyFrameCompressed, 0xA);
```

We already know the why the total size is 0xA.